chronicle-front/src/app/services/upload.service.ts

```diff
@@ -0,0 +1,68 @@
+import { HttpClient, HttpEvent, HttpHeaders, HttpRequest } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { catchError, retry } from 'rxjs/operators';
+import { Observable, throwError } from 'rxjs';
+import { analyzeAndValidateNgModules } from '@angular/compiler';
+
+@Injectable({providedIn: 'root'})
+export class UploadService {
+
+  private baseURL = 'http://localhost:8080/myapp';
```

Comment:
I foresee a bug - please refactor this to use the `environment.ts` instead of hardcoding a URL.